### PR TITLE
MIFOSX-2322

### PR DESCRIPTION
**Pull request by theupscale**
Done please test

### CODE REVIEW

File: mifosng-provider/src/main/java/org/mifosplatform/template/service/TemplateMergeService.java [hunk 1]
```diff
@@ -124,7 +129,7 @@ protected PasswordAuthentication getPasswordAuthentication() {
         try {
             connection = (HttpURLConnection) new URL(url).openConnection();
             if (this.authToken != null) {
-                connection.setRequestProperty("Authorization", "Basic " + this.authToken);
+                connection.setRequestProperty("Authorization", "Bearer " + this.authToken);
```
Comment (pramodn02) [inline mifosng-provider/src/main/java/org/mifosplatform/template/service/TemplateMergeService.java]: why this change is required?

**Comment by pramodn02** [comment]
please remove all the system out statements and also give a proper name to map

File: mifosng-provider/src/main/java/org/mifosplatform/template/service/TemplateMergeService.java (hunk 1 above)
Comment (theupscale) [inline mifosng-provider/src/main/java/org/mifosplatform/template/service/TemplateMergeService.java]: Yeah i have tested it, for both auth method

**Comment by theupscale** [comment]
Its was not working with Basic so i tested it with , Bearer and it started
workgin

On Tue, Nov 17, 2015 at 8:33 AM, pramod <notifications@github.com> wrote:

> please remove all the system out statements and also give a proper name to
> map
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openMF/mifosx/pull/1544#issuecomment-157253210>.
>



-- 

Best Regards

Saransh Sharma

Upscale Consultancy PVT LTD.

Disclaimer:

----------------------------------------------------------------------------------------------
This email was sent from within the Upscale Consultancy Services Pvt Ltd.
The contents of this email, including the attachments, are LEGALLY
PRIVILEGED AND CONFIDENTIAL to the intended recipient at the email address
to which it has been addressed. If you receive it in error, please notify
the sender immediately by return email and then permanently delete it from
your system.The unauthorized use, distribution, copying or alteration of
this email, including the attachments, is strictly forbidden. Thank
you.Please note that neither Upscale Group nor the sender accepts any
responsibility for viruses and it is your responsibility to scan the email
and attachments (if any).
----------------------------------------------------------------------------------------------
